### ISSUE TITLE
Buttons that open popups in disabled components should receive an appropriate disabled attribute

### ISSUE THREAD

**Issue by yanmariomenev** [telerik/kendo-angular#4492]
### Description

Currently, when a component like the [DateTimePicker is disabled](https://www.telerik.com/kendo-angular-ui/components/dateinputs/datepicker/disabled-state), the button that toggles the popup doesn’t have any attribute or disabled property.

Readers in cursor mode like NVDA and JAWS will read the button, but the problem is that it doesn't inform the user that the button is disabled and that they cannot interact with it:

![Image](https://github.com/user-attachments/assets/1d4071f7-6b5e-4390-99e6-49c2d8e222fb)

### Steps To Reproduce

Use NVDA in the following demo to navigate to the button - https://www.telerik.com/kendo-angular-ui/components/dateinputs/datepicker/disabled-state

### Screenshots or video

https://github.com/user-attachments/assets/38ce6cad-f9e8-4b25-8c96-a37b4ddb524c

### Actual Behavior

.

### Expected Behavior

.

### Browser

Chrome

### Browser version

latest

### OS type

MacOS
